donkeycar/utils.py

```diff
@@ -407,27 +407,42 @@ def get_model_by_type(model_type, cfg):
     if model_type == "tflite_linear":
         kl = TFLitePilot()
     elif model_type == "localizer" or cfg.TRAIN_LOCALIZER:
-        kl = KerasLocalizer(num_locations=cfg.NUM_LOCATIONS, input_shape=input_shape)
```

Comment:
Lets support the basic `linear` and `categorical` models out of the box in `get_model_by_type`.

Other models (RNN et. al.) require us to do more with the input data pipeline. We can do it as a stretch goal. Given we are not ready for this yet, I think we can leave those models in `keras.py` but not support them in `get_model_by_type` for now.

Reply:
Makes sense, let me update this.